_docs_explore/64-request-items.md

```diff
@@ -1,4 +1,277 @@
+| ----------- | ----------------------------------- | -------------------------------- |
+| `@type`     | "CreateAttributeAcceptResponseItem" | The type of the ResponseItem.    |
+| attributeId | `string`                            | The id of the created Attribute. |
+
+### Combinations and usage scenarios
+
+| Attribute Type | Attribute Owner | Possible? | Automation      | Examples/Reason                                                                                                                                                                                                                                                       |
+| -------------- | --------------- | --------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Identity       | Sender          | N         |                 | Use ShareAttributeRequestItem instead.                                                                                                                                                                                                                                |
+| Identity       | Recipient       | Y         | `USER_DECISION` | University sends student his certificate (Propose would be inappropriate in this case, because the student should not be able to return his own value)                                                                                                                |
+| Identity       | `<empty>`       | Y         |                 | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
+| Relationship   | Sender          | Y         | `AUTO_ACCEPT`   | Company sends new customer his customer number.                                                                                                                                                                                                                       |
+| Relationship   | Recipient       | Y         | `USER_DECISION` | With this combination the **sender asks the recipient for the one-time permission** to write a Relationship Attribute once AND the **sender defined a value** which can either be accepted and stored, or rejected. Thus, the user cannot change the value by itself. |
+| Relationship   | `<empty>`       | Y         |                 | An empty owner defaults to an Attribute with owner=`<recipient>`. This is needed for Requests inside of Relationship Templates, since you don’t know the Enmeshed Address of your peer before the Relationship is established.                                        |
```

Comment:
Hier müsste in der Automation Spalte derselbe Wert stehen wie bei "Attribute Owner = Recipient"